src/ui/FilterFormSummaryChips.tsx

```diff
@@ -13,6 +13,21 @@ import * as React from 'react';
 import t from 'fontoxml-localization/src/t';
 
 import useNestedCheckboxesForFilterOptions from './useNestedCheckboxesForFilterOptions';
+import { FormValueByName } from 'fontoxml-design-system/src/types';
+
+import { FormFeedbackByName } from 'fontoxml-design-system/src/types';
+import { AnnotationErrorType } from '../types';
+import { CardContentComponentProps } from 'fontoxml-feedback/src/types';
+
+type Props = {
+	error: AnnotationErrorType;
+	feedbackByName: FormFeedbackByName;
+	isDisabled: boolean;
+	isSubmitting: boolean;
+	onChange: CardContentComponentProps['onReviewAnnotationFormSubmit'];
```

Comment:
Even the `onChange` function and the submit function for the review annotation form have the same type, I think that they are not related because they do different actions, so I would set another type like `onChangeCallback` from `fontoxml-design-system` library or a custom type as `onChange(valueByName: FormValueByName): Promise<void>;`